lib/storage.ex

```diff
@@ -1,17 +1,23 @@
 defmodule Snapeth.Storage do
   require Logger
+  use Timex
+
+  def leaderboard_filename() do
```

Comment:
this filename should be memoized, I am unsure how to accomplish this in elixir. Also unfamiliar with slack bots and how often this might go down, so global variables are probably out of the question